app/src/ui/list-view/list-view.html

```diff
@@ -189,7 +189,7 @@
         </main>
 
         <pl-dialog-options id="sectionSelector" on-dialog-open="_stopPropagation"
-            on-dialog-close="_stopPropagation"></pl-dialog-options>
+            on-dialog-close="_stopPropagation" prevent-dismiss="false"></pl-dialog-options>
```

Comment:
Again, boolean html attributes shouldn't really work this way. Omitting the attribute means `false`, adding it (without a value) means `true`.

Reply:
Here is the relevant section of the specification in case you're interested: https://www.w3.org/TR/html5/infrastructure.html#boolean-attributes